tests/commerce_integration/_bootstrap.php

```diff
@@ -27,3 +31,8 @@
 
 // Disconnect Promoter to avoid license-related notices.
 remove_action( 'tribe_tickets_promoter_trigger', [ tribe( Dispatcher::class ), 'trigger' ] );
+
+// Ensure Classy is disabled for these tests.
```

Comment:
This seems like a left-over from debug.
But I would favor this approach over the skipping of tests if Classy is active.
My thinking it that any logic, be that front-end or back-end, affected by Classy work will be tested in tests dedicated to it, so it's safe for tests in these suites to run without Classy, and keep testing the current implementation, for now.